application/src/main/java/org/opentripplanner/standalone/server/GrizzlyServer.java

```diff
@@ -89,7 +90,7 @@ public void run() {
     CompressionConfig cc = httpListener.getCompressionConfig();
     cc.setCompressionMode(CompressionConfig.CompressionMode.ON);
     cc.setCompressionMinSize(50000); // the min number of bytes to compress
-    cc.setCompressableMimeTypes("application/json", "text/json"); // the mime types to compress
+    cc.setCompressibleMimeTypes("application/json", "text/json", MediaType.APPLICATION_XML); // the mime types to compress
```

Comment:
Is there constants for the two other types as well?

Reply:
28bd06a302226a03ec41f16586f7b2391fa07314

There is a constant for `application/json` but not for `text/json` because it is a very old and never specified way of saying that a resource is JSON. I personally would vote to completely remove it but lets discuss.

Reply:
We said we will remove `text/json`.